app/src/test/java/com/example/concordia_campus_guide/ConvertersTest/CoordinatesListConverterTest.java

```diff
@@ -42,6 +43,7 @@ private void setupJsonString(){
     public void convertListOfCoordinatesToStringTest(){
         String jsonObject = coordinatesListConverter.convertToDatabaseColumn(coordinatesList);
         assertEquals("convertListOfCoordinatesToStringTest:", json,jsonObject);
+        assertEquals(null,coordinatesListConverter.convertToDatabaseColumn(null));
```

Comment:
you should use assertNull instead of assertEquals(null, ...)

Reply:
Good idea 👍